### ISSUE TITLE
Demo is completely broken

### ISSUE THREAD

**Issue by ilyavolodin** [eslint/eslint.github.io#259]
Currently demo is completely broken and doesn't run at all. Not sure when it started happening. But I think maybe at this point we just want to replace it with the prototype that I have? Although, I think right now, the failure is actually in the browserified bundle. I have no experience with browserify, could somebody take a look?

cc: @eslint/eslint-team

**Comment by ilyavolodin** [comment]
Same goes for Espree demo as well. Same error. So I think it might actually be browserify issue.

**Comment by ilyavolodin** [comment]
I fixed both ESLint and Espree demo by regenerating browserify eslint.js file and pushing it to the website. It looks like what happened is @nzakas had a merge conflict when doing last release (I'm guessing that's because he didn't pull latest master for website before the release, and since I've done last 2 release, his branch was out of sync). So browserified filed contained diff with the previous version. We should probably update release script to make sure that website is on the master branch and has latest before pushing changes to it. Closing this issue, since it's now fixed.
